app/helpers/form_helper.rb

```diff
@@ -442,6 +442,15 @@ def form_select_f(f, attr, array, select_options = {}, html_options = {})
     include_blank = select_options.delete(:include_blank)
     if include_blank
       addClass html_options, "include_blank"
+      blank_value = include_blank.is_a?(TrueClass) ? nil : include_blank
+      if array.kind_of?(Array) # incase array is options_for_select
+        array = array.to_a.dup
+        blank_option = [blank_value, nil]
+        array.insert(0, blank_option)
+      end
+      html_options['data-placeholder'] = blank_value || html_options['placeholder']
+    elsif html_options[:placeholder]
+      html_options['data-placeholder'] = html_options.delete(:placeholder)
```

Comment:
On L451 we keep the `'placeholder'` key in `html_options`, but in the other branch we remove it. Is there a reason for this inconsistency?

Reply:
will fix here in both places : https://projects.theforeman.org/issues/38211